app/src/main/java/com/dimelo/sampleapp/chats/TabTelecom.java

```diff
@@ -84,12 +88,44 @@ private void customize(){
         customisation.inputbarBackgroundColor = getResources().getColor(R.color.purple_50);
         customisation.userMessageTextColor = Color.WHITE;
         customisation.agentMessageTextColor = Color.BLACK;
-        customisation.agentNameColor = Color.BLACK;
-        customisation.agentTimeColor = Color.BLACK;
+        customisation.agentNameColor = Color.RED;
+        customisation.agentTimeColor = Color.GREEN;
         customisation.systemMessageTextColor = Color.BLACK;
-        customisation.dateTextColor = Color.WHITE;
+        customisation.dateTextColor = Color.RED;
+        customisation.dateFont = Typeface.DEFAULT_BOLD;
         customisation.hourTimeTextColor = Color.WHITE;
```

Comment:
please remove this line you are already customised this one in line 117

Reply:
ok